### PR TITLE
Add status command

### PR DESCRIPTION
**Pull request by MaxGoo**
Figured a "status" command was more natural than something like "current-profile".

In the future the status could contain more relevant information in addition to the current profile

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "awsx",
   "description": "AWS CLI profile switcher with MFA support",
-  "version": "1.1.1",
+  "version": "1.1.2",
```
Comment (usernameseb) [inline package.json:4]: I think Ian's plan is to bump the version once all the hackathon changes are in and release. I might be wrong though, @iansu ?

Comment (iansu) [reply package.json:4]: Yeah, I think it makes sense to batch them together.

File: src/app.ts [hunk 2]
```diff
@@ -837,6 +837,17 @@ const awsx = (): void => {
         }
       }
     })
+    .command({
+      command: 'status',
+      describe: 'Show the status of your current awsx session',
+      handler: (): void => {
+        try {
+          console.log(chalk.green(`Current profile is '${currentProfile}'`));
```
Comment (iansu) [inline src/app.ts:845]: This is never going to throw. I think for a proper status command we should try to make some kind of "ping" request to AWS and then indicate which profile is active and if it's able to reach and authenticate with AWS.

**Review by iansu — CHANGES_REQUESTED** [review]
I think this needs to do a bit more to be a status command.

The idea behind the command to show the current profile was so that people could use it to add the name of the current profile to their shell prompt. So I think that's still worth adding in addition to a status command.

File: src/app.ts [hunk 3]
```diff
@@ -593,6 +595,37 @@ const disableMfa = async (name?: string): Promise<void> => {
...
+
+const status = async (): Promise<void> => {
+  console.log(chalk.yellow(`Checking status...`));
+
+  const sts = new STS();
+
+  const response = await Promise.race([sts.getCallerIdentity().promise(), timedOutStatusCheck()]);
+
+  if (response) {
+    try {
```
Comment (iansu) [inline src/app.ts:612]: I don't think this try/catch is doing anything. I don't think anything in the try will ever throw.

File: src/app.ts [hunk 4]
```diff
@@ -2,6 +2,8 @@ import inquirer from 'inquirer';
 import chalk from 'chalk';
 import yargs, { Argv } from 'yargs';
 import updateNotifier from 'update-notifier';
+import { STS } from 'aws-sdk';
```
Comment (iansu) [inline src/app.ts:5]: Change this to `import STS from 'aws-sdk/clients/sts'` or something to that effect. We should do this everywhere so we're not importing the entire AWS SDK.

**Review by iansu — CHANGES_REQUESTED** [review]

File: src/app.ts (hunk 4 above)
Comment (MaxGoo) [reply src/app.ts:5]: I can import from this and the types are correct, but the call just hangs - if I import from just aws-sdk it works 🤷

Comment (iansu) [reply src/app.ts:5]: That's interesting. We can try to fix this later.

**Review by iansu — APPROVED** [review]

**Review by usernameseb — APPROVED** [review]

**Pull request merged**